index.html

```diff
@@ -905,4 +959,5 @@ <h4>RAPIDOC Newsletter</h4><br>
   <script src="js/progressbar_and_header.js"></script>
   <div id="progressBar"></div>
   <script src="js/patientportal_loginjs.js"></script>
+  <!-- <script src="functions/index.js"></script> -->

```

Comment:
Why this commented?

Reply:
That file is not in use.

Reply:
Yesterday, I tried using nodemailer. However, it's not needed anymore.